students/sbolsen/lesson03/mailroom.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+
+donors = [
+        ['Jesse Johnson', [150, 345]],
+        ['Mary May', [1000, 750]],
+        ['Spencer Samuels', [50, 200, 1100]],
+        ['Zach Zillow', [85]],
+        ['Tina Thompson', [76, 250, 300]]
+        ]
+
+
+def donor_list():
+    for donor in donors:
+        print(donor[0])
+
+
+def prompt_user():
+    choice = input('Please select from the following: \n1) Send a Thank You \n2) Create a Report \n3) Quit\n')
+    return choice
+
+
+def thank_you():
+    select_user = input('Enter a full name: ')
+    if select_user == 'list':
+        donor_list()
+        select_user = input('Enter a full name: ')
+    return select_user
+
+
+def user(select_user):
+    user_exists = False
+    for item in donors:
+        for donor in item:
+            if select_user == donor:
+                user_exists = True
```

Comment:
you can just have `return` here to exit function, no need to have additional var